docs/project/introduction.mdx

```diff
@@ -27,7 +27,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 ## Introduction
 
-WeatherXM is built on an EVM compatible chain (we are currently developing on Polygon, but this will be finalized near token launch) and rewards weather station owners with the WXM token. WeatherXM intends to be the world's largest, most distributed, and most decentralized weather network.
+WeatherXM is built on an EVM compatible chain. We are currently developing on [Arbitrum One](https://arbitrum.io/), and rewards weather station owners with the WXM token. WeatherXM intends to be the world's largest, most distributed, and most decentralized weather network.
```

Comment:
Typo here, `and rewards`. I suggest keeping the initial parentheses.

Reply:
ok